test/configs/enable_verification.json

```diff
@@ -55,6 +55,12 @@
         "test/sql/function/numeric/set_seed_for_sample.test"
       ]
     },
+    {
+      "reason": "VARIANT Value internals aren't deterministic",
+      "paths": [
+        "test/sql/types/variant/implicit_cast_from_variant.test"
```

Comment:
This should be solved already, but I'm not *super* happy about the fix in terms of performance
And it's not reproducible locally, so will require a CI run to verify, which I've decided to hold off on doing until the CI is entirely green.